ZUT-PokerPlanningClient/src/app/project-planning/games/games.component.html

```diff
@@ -1 +1,4 @@
-<p>games works!</p>
+<div class="container">
```

Comment:
najs :) sprobujemy teraz to powielic dyrektywa ngFor i logika napisana w game-item.ts